### PR TITLE
Update cancel-my-account.md

### PR DESCRIPTION
**Pull request by haleysolomon**
Updated cancel and downgrade instructions for new automated workflow.

### CODE REVIEW

File: articles/faq/cancel-my-account.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
 ---
-title: "How Do I Cancel My Account?"
+title: "How Do I Downgrade or Cancel My Account?"
 description: "If you are a paying customer and you wish to cancel your account you can go into Admin Billing in your Spark Post account and click on the Free plan Your billing will automatically stop Please note If you have a customized pricing account please contact us..."
```
Comment (haroldvass) [inline articles/faq/cancel-my-account.md]: If you are a paying customer and wish to cancel your account, you can go into Admin Billing

File: articles/faq/cancel-my-account.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+If you no longer wish to use SparkPost, you have two options: (1) downgrade to the free "Test" plan, or (2) cancel your account. 
 
-Please note: If you have a customized pricing account, please [contact us.](https://www.sparkpost.com/submit-a-ticket/)
+What's the difference? 
+* Downgrading to the free "Test" plan will keep your account settings, sending domains, history, etc., and you can later choose to upgrade back to a paid plan at any time to pick up where you left off. Downgrading is recommended for "seasonal senders" who don't need to use SparkPost for several months in a row but who would like to pickup using SparkPost seamlessly in a few months.
+* Canceling your account will delete users, API keys, sending domains, etc. associated with your account, and there is no way to restore these items after cancellation. Canceling is recommended for customers who do not intend to send with SparkPost again (we are sorry to see you go!). If you cancel and later decide to use SparkPost again, you will need to create a new account and reconfigure your sending domains, settings, etc.
```
Comment (haroldvass) [inline articles/faq/cancel-my-account.md]: sending domains, etc. associated with your account, and there is no way to restore these items after cancellation.

^Split this into two sentences

File: articles/faq/cancel-my-account.md (hunk 1 above)
Comment (haleysolomon) [reply articles/faq/cancel-my-account.md]: updated the description, this was all old text actually.

File: articles/faq/cancel-my-account.md (hunk 2 above)
Comment (haleysolomon) [reply articles/faq/cancel-my-account.md]: updated.

**Pull request merged**